Core/GameResources.py

```diff
@@ -42,10 +44,42 @@ def __init__(self):
         # Add to player sprite list
         self.player_list.append(self.player_sprite)
 
+        # Enemy
+        self.enemy = Enemy(grid_x, grid_y, [self.player_sprite.center_x, self.player_sprite.center_y], my_map)
+        
+        # Add to enemy sprite list
+        self.enemy_list.append(self.enemy.enemy_sprite)
+
+        grid_size = SPRITE_SIZE
+
+        playing_field_left_boundary = -SPRITE_SIZE * 2
```

Comment:
we'll have to dynamically calculate this now that we have rooms (or the basics of rooms)